docs/platforms/javascript/common/feature-flags/index.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Set Up Feature Flags
+sidebar_order: 7000
+notSupported:
+  - javascript.aws-lambda
+  - javascript.azure-functions
+  - javascript.bun
+  - javascript.capacitor
+  - javascript.cloudflare
+  - javascript.connect
+  - javascript.cordova
+  - javascript.deno
+  - javascript.electron
+  - javascript.express
+  - javascript.fastify
+  - javascript.gcp-functions
+  - javascript.hapi
+  - javascript.koa
+  - javascript.nestjs
+  - javascript.nodejs
+  - javascript.wasm
+description: "Learn how to set up feature flag evaluation tracking and feature flag change tracking."
+---
+
+<Alert level="info" title="Currently in Beta">
+
+The support for **feature flag change tracking** and **feature flag evaluation tracking** is currently in beta.
+
+</Alert>
+
+Link your external feature flag integrations with Sentry to provide feature flag insights right inside the Sentry UI. Linking one or more integrations will allow you to view recent flag evaluations in one place and identify potential suspect flags related to errors. In addition, Sentry will provide insights on feature flag updates relative to error event timelines.
```

Comment:
"Linking one or more integrations" is it typical for folks to have more than one FF integrations? :o

Reply:
Sentry had like four different ways of handling feature flags at one point lol.  I hope its not typical but it is supported.